packages/office-addin-manifest/src/export.ts

```diff
@@ -35,9 +35,9 @@ async function createZip(manifestPath: string): Promise<AdmZip> {
     throw new Error(`The file '${manifestPath}' does not exist`);
   }
 
-  const manifest: TeamsManifestVDevPreview = (await AppManifestUtils.readTeamsManifest(
+  const manifest: DevPreviewSchema = (await AppManifestUtils.readTeamsManifest(
     manifestPath
-  )) as TeamsManifestVDevPreview;
+  )) as DevPreviewSchema;
```

Comment:
nit: if you have the type on the variable declaration then I don't think you need the `as DevPreviewSchema`.